docker.go

```diff
@@ -164,6 +167,18 @@ func getRoleArnFromEnv(env []string) (role roleArn, policy string, err error) {
 			}
 		} else if v[0] == "IAM_POLICY" && len(v) > 1 {
 			policy = strings.TrimSpace(v[1])
+		} else if v[0] == "IAM_ROLE_NAME" && len(v) > 1 {
```

Comment:
@dump247 My main reason for creating a new environment variable instead of using IAM_ROLE was because I was being conservative in not breaking existing functionality.